plugins/org.jkiss.dbeaver.ext.altibase/src/org/jkiss/dbeaver/ext/altibase/tasks/AltibaseToolReplicationStop.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.altibase.tasks;
+
+import org.jkiss.dbeaver.ext.altibase.AltibaseConstants;
+import org.jkiss.dbeaver.ext.altibase.model.AltibaseReplication;
+import org.jkiss.dbeaver.model.edit.DBEPersistAction;
+import org.jkiss.dbeaver.model.exec.DBCException;
+import org.jkiss.dbeaver.model.exec.DBCSession;
+import org.jkiss.dbeaver.model.impl.edit.SQLDatabasePersistAction;
+import org.jkiss.dbeaver.model.sql.task.SQLToolExecuteHandler;
+
+import java.util.List;
+
+public class AltibaseToolReplicationStop extends SQLToolExecuteHandler<AltibaseReplication, AltibaseToolReplicationStopSettings> {
+
+    @Override
+    public AltibaseToolReplicationStopSettings createToolSettings() {
+        return new AltibaseToolReplicationStopSettings();
+    }
+
+    @Override
+    public void generateObjectQueries(DBCSession session, AltibaseToolReplicationStopSettings settings, 
+            List<DBEPersistAction> queries, AltibaseReplication object) throws DBCException {
+        String sql = String.format(AltibaseConstants.ALTER_REPL_PREFIX + " %s STOP", object.getName());
+        queries.add(new SQLDatabasePersistAction(sql));
+    }
+
+    public boolean needsRefreshOnFinish() {
```

Comment:
```suggestion
    @Override
    public boolean needsRefreshOnFinish() {
```

Reply:
> 

Thanks. I'll apply it both AltibaseToolReplicationStart and AltibaseToolReplicationStop.